server/auth.go

```diff
@@ -368,6 +369,92 @@ func (c *client) matchesPinnedCert(tlsPinnedCerts PinnedCertSet) bool {
 	return true
 }
 
+// JWT Tag value for Account JWT Tag, enabling on-boarding services
+const featureTagOnboard = "nats-onboard-jwt"
+const onboardSubjFmt = "$jwt.request.%s"
+
+func (s *Server) onboardingCallout(c *client, opts *Options) bool {
+	var acc *Account
+	if c.opts.Account == _EMPTY_ {
+		// check default (system account) callout
+		acc = s.SystemAccount()
+		if acc == nil {
+			c.Debugf("Authentication requires a user JWT - Token Request failed - Account Missing")
+			return false
+		}
+	} else {
+		// check account listed
+		var err error
+		acc, err = s.fetchAccount(c.opts.Account)
+		if err != nil {
+			c.Debugf("Authentication requires a user JWT - Token Request Account %q lookup failed: %s", c.opts.Account, err)
+			return false
+		}
+	}
+	acc.mu.RLock()
+	signupName := fmt.Sprintf("Account: %s/%s", acc.Name, acc.nameTag)
+	featOnboard := acc.tags.Contains(featureTagOnboard)
+	acc.mu.RUnlock()
+	if !featOnboard {
+		c.Debugf("Authentication requires a user JWT")
+		return false
+	}
+	// Perform the actual JWT Request
+	msgChan := make(chan string)
+	defer close(msgChan)
+	ib := s.newRespInbox()
+	sub, err := s.sysSubscribe(ib, func(_ *subscription, _ *client, acc *Account, subject, reply string, rmsg []byte) {
+		msgChan <- string(copyBytes(rmsg))
```

Comment:
You don't need `copyBytes()` here, `string()` is already making a copy of the byte slice.

Reply:
fixed